docs/src/main/asciidoc/vertx-reference.adoc

```diff
@@ -103,10 +103,6 @@ Refer to the associated documentation to learn how to use them.
 |`io.quarkus:quarkus-mailer` (extension)
 |https://quarkus.io/guides/mailer
 
-|MQTT Client
-|`io.quarkus:quarkus-smallrye-reactive-messaging-mqtt` (extension)
-|https://quarkus.io/guides/mqtt
```

Comment:
Ah, I wasn't clear: keep the entry and just remove the link. Better than not having it.

Reply:
I replaced the link with `No guide yet` to be explicit as to why there is not link. Is this OK with you ?

Reply:
Yes.